src/plugins/formvalid/formvalid.js

```diff
@@ -102,6 +102,26 @@ var componentName = "wb-frmvld",
 						labels[ i ].innerHTML += " ";
 					}
 
+					// Hide "required" label text in older forms from screen readers
+					// Prevents redundant "required" announcements on semantically-required fields whose labels mention they're required
```

Comment:
Can you also add this note in the plugin documentation, thanks

Reply:
Done :).